src/main/java/com/igorbochkor/Main.java

```diff
@@ -14,12 +18,42 @@ public static void main(String[] args) {
         ManufacturerService manufacturerService =
                 (ManufacturerService) injector.getInstance(ManufacturerService.class);
         System.out.println(manufacturerService.get(FIRST_INDEX));
+        final Manufacturer manufacturerTesla = manufacturerService.get(FIRST_INDEX);
         System.out.println(manufacturerService.getAll());
         Manufacturer manufacturerVolvo = new Manufacturer("Volvo", "Sweden");
-        //manufacturerService.create(manufacturerVolvo);
+        manufacturerService.create(manufacturerVolvo);
         manufacturerVolvo.setCountry("Italy");
         manufacturerVolvo.setId(THIRD_INDEX);
         manufacturerService.update(manufacturerVolvo);
         manufacturerService.delete(SECOND_INDEX);
+
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Driver driverTwo = new Driver("Bob", "CE");
+        Driver driverThird = new Driver("Alice", "ABC");
+        driverService.create(driverThird);
+        System.out.println(driverService.get(SECOND_INDEX));
+        driverTwo.setId(SECOND_INDEX);
+        driverTwo.setName("Albert");
+        driverTwo.setLicenceNumber("B");
+        driverService.update(driverTwo);
+        driverService.delete(FIRST_INDEX);
+        System.out.println(driverService.getAll());
+
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        Car carOne = new Car("Model S", manufacturerTesla);
+        Car carTwo = new Car("Volvo xc90", manufacturerVolvo);
+        Car carThree = new Car("V50", manufacturerVolvo);
+        carService.create(carOne);
+        carTwo.setId(SECOND_INDEX);
+        Car car = carService.create(carThree);
+        carTwo.setManufacturer(manufacturerVolvo);
+        carService.update(carTwo);
+        carService.delete(car.getId());
+        System.out.println(carService.get(FIRST_INDEX));
```

Comment:
instead of this constants you can use `carOne.getId()` and so on

Reply:
fixed